templates/search.html

```diff
@@ -10,7 +46,13 @@ <h1 style="margin: 0 20px;">
     </a>
 </div>
 
-<p style="margin-bottom: 4px">Discover your next class with p-recs! Simply type your interests, major or learning style and find the most relevant courses for you.</p>
+<p style="margin-bottom: 4px">Discover your next class with p-recs! Simply type your interests, major or learning style and find the most relevant courses for you. 
+  <button class="btn btn-outline-secondary" data-toggle="modal" data-target="#myModal" role="button">
```

Comment:
the button looks a bit odd, i thought we discussed this as hyperlink maybe something like:
`<a href="#" data-toggle="modal" data-target="#aboutModal" role="button"> `